src/prepack-cli.js

```diff
@@ -483,6 +578,30 @@ fi
         // if it is not a FatalError, it means prepack failed, and we should display the Prepack stack trace.
         console.error(err.stack);
       }
+      if (reproOnFatal || reproUnconditionally) {
+        // Get largest list of original sources from all diagnostics.
+        // Must iterate through both because maps are ordered so we can't tell which diagnostic is most recent.
+        let largestSourceFilesList = [];
+        let largestLength = 0;
+        let sourceMaps = [];
+
+        let allDiagnostics = Array.from(compilerDiagnostics.values()).concat(compilerDiagnosticsList);
+        allDiagnostics.forEach(diagnostic => {
+          if (
+            diagnostic.sourceFilePaths &&
+            diagnostic.sourceFilePaths.sourceFiles &&
+            diagnostic.sourceFilePaths.sourceMaps
+          ) {
+            if (diagnostic.sourceFilePaths.sourceFiles.length > largestLength) {
+              largestSourceFilesList = diagnostic.sourceFilePaths.sourceFiles;
+              largestLength = diagnostic.sourceFilePaths.sourceFiles.length;
+              sourceMaps = diagnostic.sourceFilePaths.sourceMaps;
+            }
+          }
+        });
+        generateDebugRepro(sourceMaps, largestSourceFilesList);
+        return;
```

Comment:
This `return;` looks wrong. Without it, `success` would be `false`, and then we'd `process.exit(1);`. However, with the `return;`, the process is likely to return with a positive error code, even in the presence of a fatal error!

Reply:
Whoops, didn't realize that `success` controlled error code. Thanks